tests/conftest.py

```diff
@@ -80,6 +80,12 @@ def small_scenario(small_infra: Infra, foo_project_id: int, foo_study_id: int) -
     yield Scenario(foo_project_id, foo_study_id, scenario_id, small_infra.id, timetable_id)
 
 
+@pytest.fixture
+def small_scenario_v2(small_infra: Infra, foo_project_id: int, foo_study_id: int) -> Iterator[Scenario]:
```

Comment:
Why do we want an iterator?

Reply:
It's not necessary, we could return a value instead of yielding it.

I did it that way for consistency, all fixture in this file yield their results, even the ones that don't delete anything afterwards. But I can change it.

Reply:
Either way is fine, thanks